src/main/kotlin/pt/up/fe/ni/website/backend/model/Activity.kt

```diff
@@ -30,6 +30,10 @@ abstract class Activity(
     @OneToMany(fetch = FetchType.EAGER)
     open val teamMembers: MutableList<Account>,
 
+    @JoinColumn // Fixme: Don't know if this is needed
```

Comment:
This is needed in order to properly define the foreign key of the relationship (see https://www.baeldung.com/spring-data-rest-relationships for example)